app/views/signup/address.html.haml

```diff
@@ -1,44 +1,64 @@
-%h2 お届け先住所入力
+.signup-header
+  .signup-header__box
+    .signup-header__box__logo
+      =link_to '#', class: "signup-header__box__logo" do
+        = embedded_svg("fmarket_logo_red.svg", class: "signup-header__box__logo")
+  .signup-header__box__progress
+    .signup_address
+      = render partial: 'progress'
 
+.signup-main
+  .signup-main__contents
+    .signup-main__contents__top
+      発送元・お届け先住所入力
+    .address-form
+      = form_for @user do |f|
+        = fields_for :address, @user.address || Address.new do |a|
+          .signup-main__contents__form__one__group
+            .signup-main__contents__form__one__group__label
+              お名前
+              %span{"class" => "note"}必須
```

Comment:
> __[HAML-Lint]__ Style/StringHashKeys: Prefer symbols instead of strings as hash keys. (https://rubystyle.guide#symbols-as-keys) ([see on Sider](https://sider.review/gh/repos/220581877/pulls/7#issue-4848119))
>
> <details>
>
> | Rule |
> | - |
> | RuboCop |
>
> </details>

シンボル型に直しましょう

Reply:
シンボル型に直しました。
urlまで、ありがとうございます。